src/go/k8s/pkg/resources/configmap.go

```diff
@@ -355,6 +355,10 @@ func (r *ConfigMapResource) CreateConfiguration(
 		return nil, err
 	}
 
+	if r.pandaCluster.Spec.RackAwareness {
```

Comment:
given that this is just passing configuration to redpanda, i don't see why this would ever be conditional tho.

Reply:
it's not needed here. cleaned up in forced update.